.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,69 @@
+name: Bug report
+description: Report a bug in Godot Tools VS Code add-on
+body:
```

Comment:
Might want to add `labels` like the [godot-csharp-vscode](https://github.com/godotengine/godot-csharp-vscode/blob/master/.github/ISSUE_TEMPLATE/bug-report.md) repository to automatically add the `bug` label to bug reports and `enhancement` to feature requests.